packages/node/src/dev-server.ts

```diff
@@ -68,6 +73,11 @@ async function createEventHandler(
   return createServerlessEventHandler(entrypointPath, {
     shouldAddHelpers: options.shouldAddHelpers,
     useRequire,
+    launcherType: detectServerlessLauncherType(
+      entrypoint!,
+      process.versions.node
```

Comment:
This won't match the same syntax as runtime

- `process.versions.node = '16.19.0'`
- `runtime = 'nodejs16.x'`

Reply:
Indeed. `detectServerlessLauncherType()` was handling the both cases. But based on your other remark, I'll extract the version here, to pass it as a number instead.